script/cibuild

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+# script/cibuild: Setup environment for CI to run tests. This is primarily
+#                 designed to run on the continuous integration server.
+
+set -e
+
+cd "$(dirname "$0")/.."
+
+bundle exec rspec
```

Comment:
Why did the end of this change from `bundle exec rake` to `bundle exec rspec` is `rspec` just the default? I am being lazy and could probably look…

Reply:
No reason really. I might as well make it consistent though 👍

Reply:
I definately prefer to be explicit and avoid the default task and the naked `rake` expecially in scripts?